rootex/vendor/alut/COPYING

```diff
@@ -0,0 +1,483 @@
+ 
```

Comment:
This file should be renamed to `LICENSE` so that the editor/game is able to find the license file and display it.